src/styles/components/covid-banner.css

```diff
@@ -0,0 +1,183 @@
+.covid-banner {
+    display: flex;
+    flex-flow: row nowrap;
+    background-image: linear-gradient(var(--secondary-500), var(--secondary-600));
+    color: white;
+}
+
+.covid-banner-reminder {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+}
+
+.covid-warning-icon {
+    /* hide alt text if image is missing */
```

Comment:
Are these TODOs or are the margins accomplishing this?

Reply:
Hmmm, yeah I think that may have been accidentally left around. Good catch, thanks!